tests/payloads/credential-status/search-list-revocation.json

```diff
@@ -0,0 +1,28 @@
+{
+    "found": true,
+    "resource": {
+        "StatusList2021": {
+            "statusPurpose": "revocation",
+            "encodedList": "H4sIAAAAAAAAA-3BAQ0AAADCoPdPbQ8HFAAAAAAAAAAAAAAAAAAAAADwaDhDr_xcRAAA",
+            "validFrom": "2023-09-07T11:01:49.440Z"
+        },
+        "metadata": {
+            "type": "StatusList2021Revocation",
+            "encrypted": false,
+            "encoding": "base64url"
+        }
+    },
+    "resourceMetadata": {
+        "resourceURI": "did:cheqd:testnet:d4a13003-0bc5-4608-b23a-54ea90fe9f90/resources/a5dd2c2e-f884-44af-af68-ac0b8a9b4c16",
+        "resourceCollectionId": "d4a13003-0bc5-4608-b23a-54ea90fe9f90",
+        "resourceId": "a5dd2c2e-f884-44af-af68-ac0b8a9b4c16",
+        "resourceName": "cheqd-employee-credentials",
+        "resourceType": "StatusList2021Revocation",
+        "mediaType": "application/json",
+        "resourceVersion": "2023-09-07T11:01:49.443Z",
+        "created": "2023-09-07T11:01:47Z",
+        "checksum": "32ffe033eb9d6c5c0d0c84dca06a9fc0b07980e1dfc3e1bc733dc6366ca781f0",
+        "previousVersionId": "dafbe3d2-85a8-4fba-b726-a21cd4b5e381",
+        "nextVersionId": null
+    }
+}
```

Comment:
Don't do a file with `strictEquals` on the whole file. If you want, you can keep this file and then compare specific key-values against the response.